src/PhoneNumber/PhoneNumber.test.js

```diff
@@ -57,15 +57,15 @@ describe('PhoneNumber', () => {
     }
 
     clickCountry(value) {
-      this.component.find(FlagSelector).simulate('change', null, value);
+      this.component.find(PrefixSelector).simulate('change', null, value);
     }
 
-    flagSelector() {
-      return this.component.find(FlagSelector);
+    prefixSelector() {
+      return this.component.find(PrefixSelector);
     }
 
     selectedCountry() {
```

Comment:
This method is unused now, i think we should remove it

Reply:
Ok